pkg/mount/magic.go

```diff
@@ -201,7 +201,7 @@ func FSFromBlock(n string) (fs string, flags uintptr, err error) {
 		return "", 0, err
 	}
 	defer f.Close()
-	block := make([]byte, blocksize)
+	block := make([]byte, blocksize*2)
```

Comment:
what is the reason for blocksize * 2? Will this break any other file systems? I really don't know, just wondering.

Reply:
The problem is that the BTRFS magic value is inside the superblock, which is exactly 64 bytes AFTER the first blocksize (offset 65536+64 aka 0x10040), so we need to read more than 1 blocksize to determine a BTRFS file system.

Reply:
This is a very clear explanation, I wonder if you might put it in as a comment? I had no idea.